packages/amplify_auth_plugin_interface/lib/src/Session/AuthUserRequest.dart

```diff
@@ -15,7 +15,7 @@
 import './AuthUserOptions.dart';
 
 class AuthUserRequest {
```

Comment:
I think it is an uncompleted class to avoid breaking changes in the future.  As Auth methods will require the AuthUserRequest class in the future.  @haverchuck would know for sure though.